CHANGES.rst

```diff
@@ -36,6 +36,9 @@ New Features:
   sharing UI widget or other user/groups search widgets.
   [sneridagh]
 
+- Add ``available_roles`` attribute to the GET @sharing endpoint.
```

Comment:
True! 😛 Should I then not to add any changelog? Github will complain, but I guess it's bypassable.